src/plugins/data_views/server/rest_api_routes/public/has_user_data_view.ts

```diff
@@ -38,10 +42,19 @@ const hasUserDataViewRouteFactory =
     >,
     usageCollection?: UsageCounter
   ) => {
-    router.get(
+    router.versioned.get({ path, access: 'internal' }).addVersion(
```

Comment:
Versioning an internal API makes a promise that it will be stable and BWC. Any breaking changes will only be made between versions. Just checking that you're aware of it.